apps/prairielearn/elements/pl-excalidraw/pl-excalidraw.py

```diff
@@ -0,0 +1,200 @@
+import base64
```

Comment:
Looks like this file needs to be auto formatted

Reply:
done! thanks